app/controllers/payments_controller.rb

```diff
@@ -1,9 +1,12 @@
 class PaymentsController < ApplicationController
+    include PaymentsHelper
+
     skip_before_action :verify_authenticity_token
 
     def new
         @payment = Payment.new
         @order = current_order
+        binding.pry
```

Comment:
Probably another untested controller? :slightly_smiling_face:

Reply:
I'll test it right away!